tao/BUILD

```diff
@@ -1,15 +1,19 @@
 load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
 load("@local_config_tf//:build_defs.bzl", "tf_copts")
 load("@local_config_cuda//cuda:build_defs.bzl", "cuda_library", "if_cuda")
+
+# NOTE(lanbo.llb): if_dcu when building for tf serving is just a dummy impl
+load("@local_config_rocm//rocm:build_defs.bzl", "if_dcu", "if_rocm")
```

Comment:
Why DCU rules has connection with tf serving options?

Reply:
Currently not supported or even tested for dcu, just make a dummy `if_dcu` here to make this `BUILD` work without futher change about `rocm` build

Reply:
Even more confused when I see another `if_dcu` on line 19...